### DISCUSSION TITLE
Checkboxes are not being checked

### DISCUSSION THREAD

**Discussion by HudsonGraeme** [community/community#23375] [New to GitHub]
<p>Currently I am unable to check or uncheck checkboxes in task lists in GitHub issues / pull requests. It appears that when I update the state of a checkbox within a comment, a network goes through to update the markdown however the network request that’s sent does not include the changed state of the checkbox and therefore it looks like no update is made on the UI. (ex: If I check the box, it will initially show that it’s checked, then upon getting a server response it reverts back to unchecked).</p>
<p>I am pretty sure I remember being able to use checkboxes without manually going into the markdown editor to change each one’s state, but please correct me if I’m wrong on that.</p>
<p>Here are some screenshots of the issue, any help on this is greatly appreciated.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/d/9d2d4bca626e6d9330a63adea048ed7b257644af.png" data-download-href="/uploads/short-url/mqs0zq9hjH8R1bhFfctKXeQOH1l.png?dl=1" title="Screen Shot 2021-06-30 at 10.54.58 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/25019680/181076628-85ab5e23-f155-436d-8d96-ac86efd8182d.png" alt="Screen Shot 2021-06-30 at 10.54.58 AM" data-base62-sha1="mqs0zq9hjH8R1bhFfctKXeQOH1l" width="690" height="249" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/d/9d2d4bca626e6d9330a63adea048ed7b257644af_2_690x249.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/d/9d2d4bca626e6d9330a63adea048ed7b257644af_2_1035x373.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/d/9d2d4bca626e6d9330a63adea048ed7b257644af_2_1380x498.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/d/9d2d4bca626e6d9330a63adea048ed7b257644af_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-06-30 at 10.54.58 AM</span><span class="informations">2768×1001 268 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by HudsonGraeme** [comment]
<p>Here is an additional screen recording of the issue:</p>
<p><img src="https://user-images.githubusercontent.com/25019680/181076631-a8a76a08-693e-413a-9cc7-ee634a505c90.gif" alt="Checkboxes" data-base62-sha1="ibPAnrV9Io7MoRq7gVKFvF3QkwY" width="690" height="302" class="animated"></p>

**Comment by HudsonGraeme** [comment]
<p>I would really appreciate any assistance on this, I believe it may be a bug</p>

**Comment by HudsonGraeme** [comment, marked as answer]
<p>I made a support ticket with GH about this and they let me know task lists are not supported within tables, which is a bit disappointing. I’ve made a post on the GH feedback repo suggesting this change and hopefully they add it as a feature at some point because this would make testing a lot easier.</p>

**Reply by goldhoorn** [reply]
Hey @HudsonGraeme  can you let me know the ticket we have just the same problem for an enpersise solution, maybe we can escalate a bit better via our end.

**Reply by HudsonGraeme** [reply]
Hi @goldhoorn - The ticket number was
```
1235617
```

Content in the ticket was:


> Currently I am unable to check or uncheck checkboxes in task lists in GitHub issues / pull requests. It appears that when I update the state of a checkbox within a comment, a network request goes through to update the markdown however the request that’s sent does not include the changed state of the checkbox and therefore it looks like no update is made on the UI. (ex: If I check the box, it will initially show that it’s checked, then upon getting a server response it reverts back to unchecked).
>
> I am pretty sure I remember being able to use checkboxes without manually going into the markdown editor to change each one’s state, but please correct me if I’m wrong on that.
>
> Here are some screenshots of the issue, any help on this is greatly appreciated.

**Comment by ggevay** [comment]
It seems that having checkboxes inside `<details>` also doesn't work, with the same symptom. (It also breaks it for the whole issue, i.e., if there is a checkbox anywhere inside a `<details>`, then even checkboxes outside `<details>` stop working.)
